getting-started-service-registry/quickstart.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: getting-started-service-registry
+  draft: true
```

Comment:
wrong place. should be in annotations

Reply:
Looks like the contrib guide is incorrect? 
https://github.com/redhat-developer/app-services-guides/blob/main/CONTRIBUTING.adoc#draft-quick-starts

Reply:
fixed .quickstart.yml (and contrib guide) in latest commit